js/pages/characterizations/components/feature-analyses/fa-view-edit/fa-design.html

```diff
@@ -100,35 +103,37 @@
                         </div>
                     </span>
                 <!-- ko foreach: $component.data().design -->
-                <div data-bind="
-                            css: $component.classes('criteria'),
-                            eventListener: [
-                                { event: 'click', selector: '.conceptset_import', callback: $component.handleConceptSetImport},
-                                { event: 'click', selector: '.conceptset_edit', callback: $component.handleEditConceptSet }
-                            ]
-                        ">
-                    <div data-bind="css: $component.classes('criteria-heading')">
-                        <input
-                                data-bind="css: $component.classes({ element: 'criteria-name', extra: 'form-control' }), textInput: $data.name, placeholder: ko.i18n('cc.fa.criteriaName', 'Criteria name')"
-                        />
-                        <button
-                                type="button"
-                                data-bind="css: $component.classes({ element: 'criteria-delete', extra: 'btn btn-sm btn-danger' }), click: () => $component.removeCriteria($index())"
-                        ><i class="fa fa-trash-alt"></i></button>
-                    </div>
-                    <!-- Aggregate -->
-                    <div data-bind="if: $parent.data().statType() === 'DISTRIBUTION' && $data.criteriaType !== 'DemographicCriteria'">
-                        <aggregate-select params="aggregates: $component.aggregates, currentAggregate: $data.aggregate, domains: $component.domains, criteria: $data"></aggregate-select>
-                    </div>
-                    <div data-bind="if: $data.criteriaType === 'CriteriaGroup'">
-                        <criteria-group params="{ expression: { ConceptSets: $component.data().conceptSets }, group: $data.expression }"></criteria-group>
-                    </div>
-                    <div data-bind="if: $data.criteriaType === 'DemographicCriteria'">
-                        <demographic-criteria params="{ criteria: $data.expression }"></demographic-criteria>
-                    </div>
-                    <div data-bind="if: $data.criteriaType === 'WindowedCriteria'">
-                        <windowed-criteria params="{ expression: { ConceptSets: $component.data().conceptSets }, criteria: $data.expression, disableObservationPeriod: true, defaultObservationPeriod: true }"></windowed-criteria>
-                    </div>
+				<div data-bind="css: $component.canEdit() ? '' : 'feature-analysis-view-edit__content--disabled'">
```

Comment:
This CSS class `feature-analysis-view-edit__content--disabled` is defined by the parent component and I was not sure how to best use it so for now it is hardcoded.

Reply:
@alex-odysseus do you or anyone from your team provide a suggestion to handle this CSS approach?

Reply:
Just to follow up on this item: I've attempted to edit the child .less sheets to include the `disabled` BEM modifier without much luck. I'd propose just hard-coding this now.